phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -96,6 +100,30 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            // set default category icons
+            try{
```

Comment:
I think it would be better to extract the whole css creation part for a list of categories to a new private method